libs/package.json

```diff
@@ -1,23 +1,14 @@
 {
-  "name": "@audius/libs",
-  "version": "1.2.117",
```

Comment:
so is the goal to migrate all of our lib consumers to audius/sdk? epic

Reply:
Yeah that's the plan, people can either stay on deprecated `@audius/libs` or update to `@audius/sdk` and use the libs in there.

An issue with this is that the `browser` bundle doesn't include libs, so if there are people using libs in the browser right now they will have to use the `legacy` bundle. But that should work for them if they are successfully using it already